site_list.py

```diff
@@ -9,7 +9,10 @@
 
 # Removes schema-specific keywords for proper processing
 social_networks: dict = dict(data)
-social_networks.pop('$schema')
+try:
+    social_networks.pop('$schema')
+except:
+    pass
```

Comment:
Instead of using a try...except block with no error handling (just "pass"), we can pass `None` as the second argument to the .pop method. What do you think?
```suggestion
social_networks.pop('$schema', None)
```

Reply:
That is ~much~ more proper. I forgot you could do that with python.

Reply:
Also forgot I could do that from here. Oh well. It's committed regardless.